### PR TITLE
Fix unit test in PhysicsTools/UtilAlgos

### PR DESCRIPTION
**Pull request by silviodonato**
#### PR description:

This PR fixes a unit test failure due to a missing input file.

#### PR validation:

`FWLiteWithBasicAnalyzer fwliteWithPythonConfig_cfg.py`

### CODE REVIEW

File: PhysicsTools/UtilAlgos/test/fwliteWithPythonConfig_cfg.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 process = cms.PSet()
 
 process.fwliteInput = cms.PSet(
-    fileNames   = cms.vstring('root://eoscms.cern.ch//eos/cms/store/relval/CMSSW_7_2_0_pre5/RelValProdTTbar/GEN-SIM-RECO/START72_V1-v1/00000/022350A9-AC30-E411-B225-0025905A6076.root'), ## mandatory
+     fileNames   = cms.vstring('root://eoscms.cern.ch//eos/cms/store/user/cmsbuild/store/relval/CMSSW_9_3_0_pre1/RelValTTbar_13/GEN-SIM-RECO/PU25ns_92X_upgrade2017_realistic_v7-v1/00000/32EA1438-3D61-E711-8FE7-0025905B85B2.root'), ## mandatory
```
Comment (silviodonato) [inline PhysicsTools/UtilAlgos/test/fwliteWithPythonConfig_cfg.py:6]: @smuzaffar is it ok pointing directly to a file in /eos/cms/store/user/cmsbuild ?

**Comment by silviodonato** [comment]
please test

**Comment by silviodonato** [comment]
merge

File: PhysicsTools/UtilAlgos/test/fwliteWithPythonConfig_cfg.py (hunk 1 above)
Comment (mrodozov) [reply PhysicsTools/UtilAlgos/test/fwliteWithPythonConfig_cfg.py:6]: we make local copies to be used by the IBs so, yes.
If the file is not in cmsbuild and is available (somewhere) we have a job to make a local copy for the cmsbuild user.

**Comment by santocch** [comment]
+1

**Pull request merged**